src/tool/cppwinrt/cppwinrt/main.cpp

```diff
@@ -35,6 +36,8 @@ namespace xlang
         { "filter" }, // One or more prefixes to include in input (same as -include)
         { "license", 0, 0 }, // Generate license comment
         { "brackets", 0, 0 }, // Use angle brackets for #includes (defaults to quotes)
+        { "fastabi", 0, 0 }, // Enable support for the Fast ABI
```

Comment:
Out of curiosity, is there any reason to have an explicit switch for this?

Reply:
Yes. Imagine an app that wants to run everywhere from Windows 8 to Windows 10 and doesn't want to deal with conditionally supporting the fast ABI. It just wants to work everywhere with a single binary. That developer won't want to use `-fastabi`.

Reply:
Heh, I guess there's no best option no matter how you spin it. I was thinking about the producer scenario where it would be rather negative to not specify the switch.

Reply:
Right, the produce would *always* want to specify `-fastabi`. The consumer may not.